rules/saiserver.mk

```diff
@@ -0,0 +1,7 @@
+# saiserver
+
+SAISERVER = saiserver
```

Comment:
Maybe its better to pack saiserver into a .deb archive in Makefile, so it will be easier to integrate into build system